### PR TITLE
Add gao as an instrument option for apply_oe

### PR DESCRIPTION
**Pull request by NickVaughn**
Just add gao to the list of SUPPORTED_SENSORS and an `elif` clause to extract datetime and fid from a GAO/CAO radiance map. I've been meaning to do this for a while to save me having to re-implement each time I pull.

### CODE REVIEW

**Comment by pgbrodrick** [comment]
New sensors always welcome; thanks Nick!

**Comment by pgbrodrick** [comment]
@jammont - can you check the linting on this and merge in?

File: isofit/utils/apply_oe.py [hunk 1]
```diff
@@ -276,6 +277,9 @@ def apply_oe(args):
     elif args.sensor == "prisma":
         # parse flightline ID (PRISMA assumptions)
         dt = datetime.strptime(paths.fid, "%Y%m%d%H%M%S")
+    elif args.sensor == "gao":
+        # parse flightline ID (GAO/CAO assumptions)
+        dt = datetime.strptime(paths.fid[3:-5], '%Y%m%dt%H%M%S')
```
Comment (jammont) [inline isofit/utils/apply_oe.py:282]: Please replace the single quotes with double quotes in order to pass the Black linting. In order to ensure linting is automatically applied in the future, you can run a `pre-commit install` in the root directory of the project.

**Review by jammont — CHANGES_REQUESTED** [review]
Looks good! Thanks for the updates. Will merge after the one comment is resolved.

File: isofit/utils/apply_oe.py (hunk 1 above)
Comment (NickVaughn) [reply isofit/utils/apply_oe.py]: Pushed the double quote changes. Thanks!

**Pull request merged**